### PR TITLE
feat(useOnce): new function

### PR DESCRIPTION
**Pull request by icooooler**
### Additional context

---

### What is the purpose of this pull request? 

- [ ] Bug fix
- [x] New Feature
- [ ] Documentation update
- [ ] Other

### Before submitting the PR, please make sure you do the following

- [ ] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [ ] Read the [Pull Request Guidelines](https://github.com/vueuse/vueuse/blob/main/packages/guidelines.md).
- [ ] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [ ] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

**Review by songjianet — APPROVED** [review]
LGTM

File: packages/core/useOnce/index.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ref } from 'vue-demi'
+
+export const useOnce = (cb: Function) => {
+  const canUse = ref(true)
+
+  // 执行
+  const action = () => {
```
Comment (sxzz) [inline packages/core/useOnce/index.ts:7]: I think it should be a verb. How about `execute`?

File: packages/core/useOnce/index.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { ref } from 'vue-demi'
+
+export const useOnce = (cb: Function) => {
+  const canUse = ref(true)
+
+  // 执行
```
Comment (sxzz) [inline packages/core/useOnce/index.ts:6]: We should use English in an international project.

File: packages/core/useOnce/index.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { ref } from 'vue-demi'
+
+export const useOnce = (cb: Function) => {
+  const canUse = ref(true)
```
Comment (sxzz) [inline packages/core/useOnce/index.ts:4]: `executed`

File: packages/core/useOnce/index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+category: Utilities
+---
+
+# useOnce
+
+make a function can only be used once.
```
Comment (sxzz) [inline packages/core/useOnce/index.md:7]:
```suggestion
A function that can only be executed once.
```

**Review by sxzz — CHANGES_REQUESTED** [review]

**Comment by antfu** [comment]
Thanks but I don't think this is related to VueUse - you can achieve this without using Vue's reactivity. This is considered as out-of-scope.

**Pull request closed without merging**